plugins/default/documentation/editors/documentation/index.ts

```diff
@@ -29,9 +38,19 @@ function loadPlugins() {
   });
 }
 
+let navListElt = document.querySelector("nav ul");
+let mainElt =  document.querySelector("main");
+
+function openDocumentation(name: string)
+{
+  (<HTMLAnchorElement>navListElt.querySelector("li a.active")).classList.remove("active");
+  (<HTMLElement>mainElt.querySelector("article.active")).classList.remove("active");
```

Comment:
We've decided to use the `as` operator rather than chevrons, like so: `(navListElt.querySelector("li a.active") as HTMLAnchorElement). ...`